### PR TITLE
Ddf-1938 Restructure documentation into 

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?
Restructures documentation modules to publish all Management, Integration, and Extending information as complete documents. 
#### Who is reviewing it?
@ahoffer @stustison @pklinef @oscaritoro @brianfelix @mcalcote @vinamartin @coyotesqrl @shaundmorris 
#### How should this be tested?
builds successfully, generates and installs appropriate documentation.
#### Any background context you want to provide?
It is no longer necessary to render separate app documentation. A possible next step may be to combine all documentation into a single module.
#### What are the relevant tickets?
DDF-1938
#### Screenshots (if appropriate)
n/a
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -96,6 +96,7 @@
         <ddf-branding>DDF</ddf-branding>
         <ddf-branding-lowercase>ddf</ddf-branding-lowercase>
         <ddf-branding-expanded>Distributed Data Framework</ddf-branding-expanded>
+        <ddf.version>${project.version}</ddf.version>
         <!--Needed for documentation formatting-->
         <at-symbol>@</at-symbol>
```
Comment (oscaritoro) [inline pom.xml]: Isn't this a little overkill?

**Comment by oscaritoro** [comment]
LGTM :+1: 
And in case you haven't got a hero, I was able to build DDF without issues (to look at the generated docs).

**Comment by coyotesqrl** [comment]
LGTM

**Comment by brianfelix** [comment]
LGTM :+1:

File: pom.xml (hunk 1 above)
Comment (vinamartin) [inline pom.xml]: I was also wondering why this substitution is needed. Any insight, @ricklarsen ? :)

**Comment by vinamartin** [comment]
:shipit:

File: pom.xml (hunk 1 above)
Comment (ricklarsen) [inline pom.xml]: having a literal `@` symbol will escape the rest of the substitutions on the same line. It's a workaround.

Comment (stustison) [inline pom.xml]: You might want to add that info that it is a workaround to the comment above it. That seems like something someone would "fix" by getting rid of it, thereby breaking the documentation.

Comment (ricklarsen) [inline pom.xml]: That's a good point. I'll add a comment.

**Comment by ryeats** [comment]
:+1: :+1:  heroed

**Pull request merged**